src/Command/VendorCommand.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace PhpSerializers\Benchmarks\Command;
+
+use PhpBench\Benchmark\BenchmarkFinder;
+use PhpBench\Benchmark\Metadata\BenchmarkMetadata;
+use PhpSerializers\Benchmarks\AbstractBench;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @author Tales Santos <tales.augusto.santos@gmail.com>
+ */
+class VendorCommand extends Command
+{
+    /**
+     * @var BenchmarkFinder
+     */
+    private $finder;
+
+    /**
+     * @var string
+     */
+    private $path;
+
+    /**
+     * @var array
+     */
+    private $packages;
+
+    /**
+     * VendorCommand constructor.
+     * @param BenchmarkFinder $finder
+     * @param string $path
+     */
+    public function __construct(BenchmarkFinder $finder, string $path)
+    {
+        parent::__construct();
+        $this->finder = $finder;
+        $this->path = $path;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('vendors')
+            ->setDescription('List all available serializers and its current version');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        if (!file_exists($lockFile = __DIR__ . '/../../composer.lock')) {
+            throw new \RuntimeException(
+                'File composer.lock was not found. You should install the dependencies before running this command'
+            );
+        }
+
+        $this->packages = json_decode(file_get_contents($lockFile), true);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $benchmarks = $this->finder->findBenchmarks($this->path);
+
+        $rows = [];
+
+        /** @var BenchmarkMetadata $benchmark */
+        foreach ($benchmarks as $benchmark) {
+
+            $ref = new \ReflectionClass($benchmark->getClass());
+
+            if (!$ref->isSubclassOf(AbstractBench::class)) {
+                continue;
+            }
+
+            $instance = $ref->newInstanceWithoutConstructor();
+
+            $rows[] = [$ref->getShortName(), $this->getVersion($instance->getPackageName())];
+        }
+
+        $style = new SymfonyStyle($input, $output);
+        $style->table(['name', 'version'], $rows);
+    }
+
+    private function getVersion(?string $package): string
```

Comment:
not sure if this makes sense as it will not be easy to test multiple versions of the same serializer... will require huge re-work of this benchmark

Reply:
I will prefer to have `name`, `description`. Have a optional method `getDescription` the serializer could add some extra info about the test, (as type, normalizes, groups... or anything else the author might like)

Reply:
What do you mean with `multiple version of the same serialize`? JMS has two implementations and will be listed twice with the same version.

Reply:
I agre, but showing something as 

```
jms/serializer 2.0.0
jms/serializer 2.0.0
```
has no value, it should be something as

```
jms/serializer 
jms/serializer minimal
```

Having the version could be hard, and some serializers (`tsantos/serializer` as example) are dev versions

Reply:
Of course if is possible to get the version, i'm fine with that, but we need some kind of "note" to distinguish variations of the same project

Reply:
Maybe we can group same vendor in one row:

<table>
<tr>
<th> Bench Name</th>
<th>Version</th>
<th>Note</th>
</tr>
<tr>
<td>JMS</td>
<td rowspan=2>2.0.0</td>
<td>Standard usage of JMS Serializer</td>
</tr>
<tr>
<td>JMS Minimal</td>
<td>JMS without default handlers</td>
</tr>
<tr>
<td>TSantos</td>
<td>dev-jms-metadata-v2</td>
<td>Standard usage of TSantos Serializer</td>
</tr>
</table>